operators/rename_from_scene_strip.py

```diff
@@ -0,0 +1,47 @@
+import bpy
+import operator
+
+from bpy.props import StringProperty
+from .utils.doc import doc_name, doc_idname, doc_brief, doc_description
+
+class RenameStripScene(bpy.types.Operator):
+    """
+    Rename a Scene Strip and its source scene.
+    """
+    doc = {
+        'name': doc_name(__qualname__),
+        'demo': '',
+        'description': doc_description(__doc__),
+        'shortcuts': [],
+        'keymap': 'Sequencer'
+    }
+
+    bl_idname = doc_idname(doc['name'])
+    bl_label = doc['name']
+    bl_description = doc_brief(doc['description'])
+    bl_options = {"REGISTER", "UNDO"}
+
+    new_name = StringProperty(
+        name="Strip New Name",
+        description="The name both the SceneStrip and its source Scene will take",
+        default="")
+
+    @classmethod
+    def poll(cls, context):
+        is_all_scene = True
+
+        for sequence in context.selected_sequences:
+            if not sequence.type == 'SCENE':
+                is_all_scene = False
+        return is_all_scene and context.selected_sequences != None
```

Comment:
hm, thought I already commented on this. So this `!= None` check doesn't make sense because if `context.selected_sequences` would be `None` it would already give an error at the beginning of the `for` loop since you can't iterate over `None`. Also use `obj is None` and `obj is not None`. It considered bad  code to have `obj == None` or `obj != None` cause `None` is a singleton value and it's more towards plain english the other way :)